<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LDFCLusterPropsDlg</class>
 <widget class="QDialog" name="LDFCLusterPropsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LIN Cluster properties</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="editProtocolVer"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>kbps</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>108</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>LIN Langauge Version</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>108</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>LIN Baud Rate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>108</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>LIN Protocol Version</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBaudRate">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>2.400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9.600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10.417</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19.200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editLangaugeVer"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelChannelName">
         <property name="text">
          <string>Channel Name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editChannelName"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonOption">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
